client/src/App.tsx

```diff
@@ -4,7 +4,8 @@ import { BrowserRouter, Route, Redirect, Switch } from 'react-router-dom';
 import Login from './pages/Login/Login';
 import Signup from './pages/SignUp/SignUp';
 import Dashboard from './pages/Dashboard/Dashboard';
-import { AuthProvider } from './context/useAuthContext';
+import BoardUI from './components/BoardUI/BoardUI';
+// import { AuthProvider } from './context/useAuthContext';
```

Comment:
commented out code here?

Reply:
Commented out AuthProvider to prevent re-directs to Login page so that team can view Board component.

Should I add it back?

Reply:
When pushing code to main we want to avoid commented out code. If it needs to be commented it can be removed :) We can always add it back in later if we need to.
In this case I would say you should uncomment useAuth. Remember that main is where all your teammates will also be pulling from, so we want to make sure it is the most up to date running version of our app!